voicer-client/src/components/marketplace/Marketplace.jsx

```diff
@@ -1,71 +1,102 @@
-import React, { useState, useContext, useEffect } from "react";
-import { useParams } from "react-router-dom";
-import { DataContext } from "../../context/DataContext";
-import axios from "axios";
-import JobsCard from "./JobsCard";
-import { jobs } from "../../fakedata/jobs";
+import React, { useState, useContext, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { DataContext } from "../../context/DataContext"
+import axios from "axios"
+import JobsCard from "./JobsCard"
+// import { jobs } from "../../fakedata/jobs"
 
-import Hero from "../hero/Hero";
+import Hero from "../hero/Hero"
 
 export default function Marketplace() {
-  const [jobMatchesDB, setJobMatchesDB] = useState(true);
-  const [data, setData] = useState([]);
-  const { token, url } = useContext(DataContext);
+  const [jobMatchesDB, setJobMatchesDB] = useState(true)
+  const [data, setData] = useState([])
+  const { token, url } = useContext(DataContext)
 
-  const jobId = useParams().jobId;
-  console.log(jobId);
+  const jobId = useParams().jobId
 
+  useEffect(() => {
+    if (jobId) {
+      console.log("we are in 'detailed job mode")
+      axios
```

Comment:
we'll eventually need to update this to axios with auth, then get the token that's returned.